shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -367,7 +367,10 @@ - (void)setSplashScreenView:(UIView*)view {
 }
 
 - (void)setFlutterViewDidRenderCallback:(void (^)(void))callback {
-  _flutterViewRenderedCallback.reset(callback, fml::OwnershipPolicy::Retain);
```

Comment:
Is this meant to be part of this change?

I think this is confusing since it does nothing if the platformView isn't available yet. (Or if that's impossible, we should assert that.)

(And if we do replace _flutterViewRenderedCallback here, then I think it can be removed entirely.)

Reply:
This should probably be moved out to a separate change.  I was experimenting with it and meant to reset it.  This is currently broken though.